src/Hocon.Configuration/Config.cs

```diff
@@ -93,20 +111,17 @@ protected Config Copy()
 
         protected override HoconValue GetNode(HoconPath path, bool throwIfNotFound = false)
         {
-            HoconValue result;
             try
             {
-                result = Root.GetObject().GetValue(path);
+                return Root.GetObject().GetValue(path);
             }
             catch
             {
                 if (throwIfNotFound)
                     throw;
 
-                result = Fallback?.GetNode(path);
```

Comment:
No need to check for fallback here anymore - if `Root` does not have it, then it just does not exist

Reply:
Much better - I like this change. Makes sense.